.travis.yml

```diff
@@ -5,6 +5,7 @@ php:
   - 7.3
   - 7.4
   - 8.0
+  - 8.1.0
```

Comment:
Why is this 8.1.0? Why not just 8.1? 8.1.4 is out already...

Reply:
This is a Travis issue. Afaik only works with 8.1.0. 
For example [8.1.3](https://app.travis-ci.com/github/wmde/Diff/jobs/563855865)

Reply:
Aha, ok then. Annoying, but hopefully something they fix later...

Reply:
https://travis-ci.community/t/php-8-1-support/12439/23 for future reference

https://docs.travis-ci.com/user/languages/php/#php-versions seems to suggest there is a 8.1 and an 8.1.4 (but no 8.1.3)..

Reply:
Correction. There's `8.1` on 16.04... Not so helpful.

There seems to be a mismatch of random PHP versions on different Ubuntu releases. 20.04 and 18.04 have `8.1.0` and `8.1.2`. 16.04 has `8.1`, `8.1.0`, `8.1.2`, `8.1.4`.

Might be "slightly" better to use `8.1.2` as the latest point release available.